src/keypair.rs

```diff
@@ -0,0 +1,69 @@
+use algebra::{ProjectiveCurve, UniformRand};
+
+use crate::domain::*;
+
+use crate::pubkey::*;
```

Comment:
in general, you should avoid importing everything (`::*`), also you can combine these with `use crate::{domain, pubkey};`

Reply:
Done!